Simplified/OPDS/NYPLOPDSAcquisition.h

```diff
@@ -100,4 +100,6 @@ NYPLOPDSAcquisitionRelationString(NYPLOPDSAcquisitionRelation relation);
 /// @c acquisitionWithDictionary: method for later deserialization.
 - (NSDictionary *_Nonnull)dictionaryRepresentation;
 
+- (void)updateAvailability:(id<NYPLOPDSAcquisitionAvailability> _Nonnull)availability;
```

Comment:
instead of adding this specific implementation, why not just removing the `readonly` modifier from line 75: 
```
@property (nonatomic, readonly, nonnull) id<NYPLOPDSAcquisitionAvailability> availability;
```